src/status_im/contexts/wallet/swap/set_spending_cap/view.cljs

```diff
@@ -0,0 +1,254 @@
+  [quo/data-item
+   {:container-style style/detail-item
+    :blur?           false
+    :card?           false
+    :network-image   network-image
+    :subtitle-type   (if network-image :network :default)
+    :status          (if loading? :loading :default)
+    :title           title
+    :subtitle        subtitle
+    :size            size}])
+
+(defn- transaction-details
+  [{:keys [estimated-time-min max-fees network loading-fees?]}]
+  [rn/view {:style style/details-container}
+   [:<>
+    [data-item
+     {:title         (i18n/label :t/network)
+      :subtitle      (:full-name network)
+      :network-image (:source network)}]
+    [data-item
+     {:title    (i18n/label :t/est-time)
+      :subtitle (i18n/label :t/time-in-mins {:minutes (str estimated-time-min)})}]
+    [data-item
+     {:title    (i18n/label :t/max-fees)
+      :subtitle max-fees
+      :loading? loading-fees?
+      :size     :small}]]])
+
+(defn footer
+  [{:keys [estimated-time-min native-currency-symbol network theme account-color loading-fees?]}]
+  (let [native-token  (when native-currency-symbol
+                        (rf/sub [:wallet/token-by-symbol
+                                 native-currency-symbol]))
+        fee-formatted (rf/sub [:wallet/wallet-send-fee-fiat-formatted
+                               native-token])]
+    [rn/view {:style {:margin-bottom -10}}
+     [transaction-details
+      {:estimated-time-min estimated-time-min
+       :max-fees           fee-formatted
+       :network            network
+       :loading-fees?      loading-fees?
+       :theme              theme}]
+     [standard-auth/slide-button
+      {:size                :size-48
+       :track-text          (i18n/label :t/slide-to-swap)
+       :container-style     {:z-index 2}
+       :customization-color account-color
+       :disabled?           loading-fees?
+       :auth-button-label   (i18n/label :t/confirm)}]]))
+
+(defn view
+  []
+  (let [theme                   (quo.theme/use-theme)
+        swap-transaction-data   (rf/sub [:wallet/swap])
+        {:keys [asset-to-pay network pay-amount
+                providers swap-proposal
+                loading-fees?]} swap-transaction-data
+        estimated-time-min      (:estimated-time swap-proposal)
+        pay-token-symbol        (:symbol asset-to-pay)
+        pay-token-address       (:address asset-to-pay)
+        native-currency-symbol  (get-in swap-proposal [:from :native-currency-symbol])
+        account                 (rf/sub [:wallet/current-viewing-account])
+        account-color           (:color account)
+        provider                (first providers)]
+    [rn/view {:style style/container}
+     [floating-button-page/view
+      {:footer-container-padding 0
+       :header                   [quo/page-nav
+                                  {:icon-name           :i/close
+                                   :on-press            on-close-action
```

Comment:
```suggestion
                                   :on-press            navigation/navigate-back
```
from this package          [utils.navigation :as navigation]

Reply:
Fixed!